Views/Parts/Product.AddToWishlistButton.cshtml

```diff
@@ -0,0 +1,60 @@
+﻿@using Nwazet.Commerce.Models
+@using Orchard.ContentManagement
+
+@using Orchard.Security
+@{
+    int productId = (int)Model.ProductId;
+    List<WishListListPart> wishLists = (List<WishListListPart>)Model.WishLists;
+    IUser user = (IUser)Model.User;
+
+    Script.Require("jQuery");
+    Style.Include("wishlists.nwazet-commerce.css", "wishlists.nwazet-commerce.min.css");
+
+    var addToWishlistText = T("Add to WishList").Text;
+}
+
+@using (Html.BeginFormAntiForgeryPost(
+                                        Url.Action("AddToWishList", "WishLists", new { area = "Nwazet.Commerce" }),
+                                        FormMethod.Post,
+                                        new { enctype = "multipart/form-data" }
+                                        )) {
+
+    if (user != null) {
+        var wishlistItems = new List<SelectListItem>();
+        wishlistItems.Add(new SelectListItem {
+            Value = "0",
+            Text = T("WishList (default)").Text
+        });
+        wishlistItems.AddRange(wishLists.Select(s => new SelectListItem {
+            Value = s.Id.ToString(),
+            Text = Html.ItemDisplayText((IContent)(s.ContentItem)).ToString()
+        }));
+        @Html.DropDownList("wishListid", wishlistItems, new { @class = "wish-list-dropdownlist" })
+        <input type="hidden" id="productid" name="productid" value="@productId" />
+        <button id="wishlist-add" type="submit" name="wishlist-add">
+            @T("Add to wishlist")
+        </button>
+        if (Model.AttributeShapes != null) {
+            <div id="add-to-wishlist-attributes" style="display:none;">
+                @*Attributes will be copied in this div once the button is clicked*@
+            </div>
+        }
+    }
+
+}
+
+@using (Script.Foot()) {
+    <script type="text/javascript">
+        jQuery(function ($) {
+            $('#wishlist-add').click(function () {
```

Comment:
There's a lot of elements in here that use a statically assigned, fixed id. Even if we think there will ever be one of those on the page, this could be made cleaner using `Id` helpers that MVC provides. Also, whenever you have a script like that, it should be in a separate script file. Inline scripts in the view should only be initializing properties to dynamic values.